perception/autoware_multi_object_tracker/src/multi_object_tracker_node.cpp

```diff
@@ -221,21 +224,30 @@ void MultiObjectTracker::onTimer()
 {
   const rclcpp::Time current_time = this->now();
 
-  // get objects from the input manager and run process
-  ObjectsList objects_list;
-  const bool is_objects_ready = input_manager_->getObjects(current_time, objects_list);
-  if (is_objects_ready) {
-    onMessage(objects_list);
+  // ensure minimum interval: room for the next process(prediction)
+  const double minimum_publish_interval = publisher_period_ * 0.85;  // 85% of the period
```

Comment:
[imo]
How about declare these magic number in other place?
Like adding following parameter in the node deceleration.

```c++
  static constexpr double minimumPublishIntervalRatio = 0.85;
  static constexpr double maximumPublishIntervalRatio = 1.05;
```

Reply:
resolved https://github.com/autowarefoundation/autoware.universe/pull/8657/commits/ff8b6464e48b21f9a0743ce796e5a2b358c2482a